cmd/insolar/certgen.go

```diff
@@ -191,9 +210,14 @@ func (g *certGen) getNodeRefByPk() insolar.Reference {
 
 	keySerialized, err := g.keyProcessor.ExportPublicKeyPEM(g.privKey)
 	checkError("Failed to export public key:", err)
-	request := requester.RequestConfigJSON{
-		Method: "GetNodeRef",
-		Params: []interface{}{keySerialized},
+	request := requester.Request{
+		JSONRPC: JSONRPCVersion,
+		ID:      1,
+		Method:  "api.call",
+		Params: requester.Params{
+			CallSite:   "contract.getNodeRef",
```

Comment:
This is the only place with name `contract.getNodeRef`, in contracts and tests we use name `GetNodeRef`. Use one of them everywhere.